api/src/test/java/org/openmrs/validator/AllergyValidatorTest.java

```diff
@@ -38,10 +38,14 @@
 import org.openmrs.Allergen;
 import org.openmrs.AllergenType;
 import org.openmrs.Allergies;
+import org.openmrs.util.OpenmrsConstants;
+import org.openmrs.api.context.Context;
+import org.openmrs.api.AdministrationService;
+import org.openmrs.test.BaseContextSensitiveTest;
 
-@RunWith(PowerMockRunner.class)
-@PrepareForTest(Context.class)
```

Comment:
Mocked tests run faster than context sensitive ones. Do you have any reason for changing this?